File: Amigo.Tenant.UI.Web/src/app/shared/dtos/equipmentStatusDTO.ts
import {Component} from '@angular/core';

export class EquipmentStatusDTO {

    equipmentStatusId: number;
    name: string;
    code: string;
    rowStatus: string;

    constructor(private equipmentStatusId1: number, name1: string) {
        this.equipmentStatusId = equipmentStatusId1;
        this.name = name1;
    }

}

File: Amigo.Tenant.Application.DTOs/Responses/Tracking/EquipmentStatusDTO.cs

namespace Amigo.Tenant.Application.DTOs.Responses.Tracking
{
    public class EquipmentStatusDTO: IEntity
    {
    public int EquipmentStatusId { get; set; }
    public string Name { get; set; }
    public string Code { get; set; }
    public bool RowStatus { get; set; }

}
}
